### PR TITLE
DO NOT MERGE [DEVELOPER-5529] Connectors no DCP

### PR DESCRIPTION
**Pull request by jordanpagewhite**
### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5574
* https://issues.jboss.org/browse/DEVELOPER-5529

### Verification Process

* Go to the connectors page (/products/fuse/connectors)
* Verify no calls are made to the DCP occur on load
* Verify no calls are made to the DCP on sort and ordering
* Verify sorting is working as expected
* Verify images display for each connectors

### CODE REVIEW

**Comment by jordanpagewhite** [comment]
This is blocked by https://issues.jboss.org/browse/DEVELOPER-5422

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/scripts/@rhd/js/namespace.js [hunk 1]
```diff
@@ -19,21 +19,6 @@ app.downloadManagerBaseUrl = "#{site.download_manager_base_url}";
...
-app.templates.searchpageTemplate = '#{partial "searchpage.html.slim"}';
-app.templates.miniBuzzTemplate = '#{partial "mini_buzz_template.html.slim"}';
-app.templates.productBuzzTemplate = '#{partial "product_buzz_template.html.slim"}';
-app.templates.buzzTemplate = '#{partial "buzz_template.html.slim"}';
-app.templates.termsAndConditionsTemplate = '#{partial "terms_and_conditions.html.slim"}';
-app.templates.bookTemplate = '#{partial "book.html.slim"}';
-app.templates.connectorTemplate = '#{partial "product-connectors-item.html.slim"}';
-
-app.templates.productStackoverflowTemplate = '#{partial "product_stackoverflow_template.html.slim"}';
-app.templates.stackoverflowTemplate = '#{partial "stackoverflow_template.html.slim"}';
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/scripts/@rhd/js/namespace.js]: I don't think this file needs to stay anymore. It's all in drupal-namespace.js I think.

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/scripts/@rhd/js/connectors.js [hunk 2]
```diff
@@ -90,9 +90,11 @@ app.connectors = {
      * @param {Array.<string>=} featuredIDs
      */
     connectorFilter : function(query, container, target_product, thumbnailSize, orderBy, featuredIDs) {
-        var url = app.dcp.url.connectors;
+        var url = '/drupal/connectors';
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/scripts/@rhd/js/connectors.js]: Please explain to me how this is actually going to work with the export.

**Comment by jordanpagewhite** [comment]
# NOTE

This PR should probably be considered Blocked until we have images uploaded to Drupal for all of the connectors. I have already discussed this with SJ.

**Comment by LightGuard** [comment]
Are we keeping this one around, or doing a new PR?

**Pull request merged**